<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="3" passed="2">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="8151" started-at="2019-07-23T07:50:29Z" finished-at="2019-07-23T07:50:37Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="8151" started-at="2019-07-23T07:50:29Z" finished-at="2019-07-23T07:50:37Z">
      <class name="testcases.UAT_RegisterPageTestCanada">
        <test-method status="PASS" signature="launchBrowser()[pri:0, instance:testcases.UAT_RegisterPageTestCanada@2c7db926]" name="launchBrowser" is-config="true" duration-ms="5463" started-at="2019-07-23T08:50:24Z" finished-at="2019-07-23T08:50:29Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchBrowser -->
        <test-method status="PASS" signature="setUp()[pri:0, instance:testcases.UAT_RegisterPageTestCanada@2c7db926]" name="setUp" is-config="true" duration-ms="87" started-at="2019-07-23T08:50:29Z" finished-at="2019-07-23T08:50:29Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="PASS" signature="loginToSite()[pri:1, instance:testcases.UAT_RegisterPageTestCanada@2c7db926]" name="loginToSite" duration-ms="22" started-at="2019-07-23T08:50:29Z" finished-at="2019-07-23T08:50:29Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginToSite -->
        <test-method status="PASS" signature="enterDetails()[pri:2, instance:testcases.UAT_RegisterPageTestCanada@2c7db926]" name="enterDetails" duration-ms="539" started-at="2019-07-23T08:50:29Z" finished-at="2019-07-23T08:50:30Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- enterDetails -->
        <test-method status="FAIL" signature="enterForm()[pri:0, instance:testcases.UAT_RegisterPageTestCanada@2c7db926]" name="enterForm" duration-ms="7483" started-at="2019-07-23T08:50:30Z" depends-on-methods="testcases.UAT_RegisterPageTestCanada.enterDetails" finished-at="2019-07-23T08:50:37Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: null
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.fail(Assert.java:101)
	at testcases.UAT_RegisterPageTestCanada.enterForm(UAT_RegisterPageTestCanada.java:304)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- enterForm -->
        <test-method status="PASS" signature="afterTest()[pri:0, instance:testcases.UAT_RegisterPageTestCanada@2c7db926]" name="afterTest" is-config="true" duration-ms="1" started-at="2019-07-23T08:50:37Z" finished-at="2019-07-23T08:50:37Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterTest -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:testcases.UAT_RegisterPageTestCanada@2c7db926]" name="tearDown" is-config="true" duration-ms="387" started-at="2019-07-23T08:50:38Z" finished-at="2019-07-23T08:50:38Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- testcases.UAT_RegisterPageTestCanada -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
